pallets/simple-nft/src/lib.rs

```diff
@@ -109,7 +111,7 @@ pub mod pallet {
         pub(super) fn run_process(
```

Comment:
need to regenerate weights for `run_process`. I think this is described inthe readme 🤔

Reply:
done